#ifndef __mm_basic_frame_control_s_control_rs_h__
#define __mm_basic_frame_control_s_control_rs_h__

#include "core/mm_core.h"

//////////////////////////////////////////////////////////////////////////
extern void hd_s_control_get_logger_rs(void* obj,void* u,struct mm_packet* rs_pack);
extern void hd_s_control_set_logger_rs(void* obj,void* u,struct mm_packet* rs_pack);
//////////////////////////////////////////////////////////////////////////

#endif//__mm_basic_frame_control_s_control_rs_h__